app-backend/common/src/main/scala/Authentication.scala

```diff
@@ -76,8 +91,14 @@ trait Authentication extends Directives {
         val picture = jwtClaims.getStringClaim("picture")
         onSuccess(UserDao.getUserById(userId).transact(xa).unsafeToFuture).flatMap {
           case Some(user) => {
-
-            val updatedUser = user.copy(email = email, name = name, profileImageUri = picture)
+            val (orgDropboxCredential, orgPlanetCredential) = getOrgCredentials(user)
+            val updatedUser = (user.dropboxCredential, user.planetCredential) match {
+              case (Credential(Some(d)), Credential(Some(p))) if d.length() == 0 =>
+                user.copy(email = email, name = name, profileImageUri = picture, dropboxCredential = orgDropboxCredential)
```

Comment:
It might be more legible to copy `email`, `name`, and `profileImageUri` first and then copy credentials onto that

Reply:
~I think this pattern will short-circuit setting the planet credential in cases where both credentials are present on the user and both have 0 length~ welp I was wrong :tada: